addons/test_discuss_full/tests/test_performance.py

```diff
@@ -21,10 +21,10 @@ class TestDiscussFullPerformance(HttpCase):
     #     4: settings
     #     1: has_access_livechat
     _query_count_init_store = 17
-    _query_count = 50 + 1  # +1 is necessary to fix nondeterministic issue on runbot
+    _query_count = 50 + 1 + 3  # +1 is necessary to fix nondeterministic issue on runbot, +3 is for the check_access_rule in _compute_is_editable
```

Comment:
You can write 53 directly, +1 is a specific case so we don't forget.
If you measured 54, you actually need to write 54 and keep the +1 too.

You can add a comment above the counter similar to others to document 3 are coming from there.